modules/generator-primer-module/README.md

```diff
@@ -0,0 +1,107 @@
+# Primer CSS Module Generator
```

Comment:
We prefix all our other module names with primer, can you rename this to `primer-module-generator`?

Reply:
I like the idea of having a separate directory for tools 👍  - would you class storybook in that category too?

Reply:
I'm 👍 for moving this into a tools directory, unsure if you want to do this as well as moving `primer-module-build` in this pr though?